### ISSUE TITLE
SA1500 is producing a false positive

### ISSUE THREAD

**Issue by vweijsters** [DotNetAnalyzers/StyleCopAnalyzers#2632]
from @JimBobSquarePants (in #2607):

I'm still seeing this for multiline arrays in 1.1.0-beta006

e.g.

```C#
        private static readonly float[,] AtkinsonMatrix =
            new float[,]
            {
               { 0, 0, 1, 1 },
               { 1, 1, 1, 0 },
               { 0, 1, 0, 0 }
            };
```

This construction works fine in 1.1.0-beta004 and no warnings should be raised here.
